### DISCUSSION TITLE
Developer who does not have access to the repository is committing changes

### DISCUSSION THREAD

**Discussion by nexusz** [community/community#84307] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have several developers working on private repositories.
Since beginning of the week I see the name of one developer appearing on commits on a project he does not have access to.
I checked the commit and it contains code that cannot be written by that person.

I suspect the the developer who is writing the code somehow is committing changes in name of somebody else.
The 2 developers do not know each other for sure.

Any idea what can cause this?

**Comment by darkotodoric** [comment]
Probably one of the developers has a virus on the computer that pushes the code into the repositories it finds on the computer.
Can you share with me the part of the code that was inserted? Maybe I can find something more if I see what is included.

I don't know how you added developers to have access to that repository, but if it's via SSH key (not via GitHub collaborators), it would be good to change it.
